### PR TITLE
solara trading sugarscape: Use coord_iter

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

**Comment by rht** [comment]
This is preferable because `model.grid._grid` is an internal attribute, and its direct access should be discouraged.

**Comment by rht** [comment]
2023-07-07-101500_1210x568_scrot
More realistic example of what it looks like.

File: examples/sugarscape_g1mt/app.py [hunk 1]
```diff
@@ -15,20 +15,19 @@ def portray(g):
...
 
-        for i in range(g.width):
-            for j in range(g.height):
-                content = g._grid[i][j]
-                for agent in content:
-                    if isinstance(agent, Trader):
-                        layers["trader"]["x"].append(i)
-                        layers["trader"]["y"].append(j)
+        # TODO update to Mesa 2.0 API
+        for content, i, j in g.coord_iter():
```
Comment (tpike3) [inline examples/sugarscape_g1mt/app.py:19]: Thanks @Corvince !

**Review by tpike3 — APPROVED** [review]
LGTM

**Pull request merged**